src/components/Card.tsx

```diff
@@ -57,17 +58,19 @@ const getProgress = (
 };
 
 const Card = (props: CardProps) => {
-  const { title, shortName, logo, currentSeason, nextSeason, seasonKeyword } =
+  const { title, shortName, logo, url, currentSeason, nextSeason, seasonKeyword } =
     props;
 
   return (
     <section className="p-6 rounded-md flex flex-col gap-2 md:gap-4 bg-gray-800">
       <div className="relative flex flex-row justify-center h-[100px] w-[140px] max-h-[100px] md:h-[140px] md:w-[200px] md:max-h-[140px] place-self-center">
-        <GatsbyImage
-          image={getImage(logo!)!}
-          alt={`${title} logo`}
-          className="my-auto"
-        />
+          <a href={url} rel="nofollow noreferrer" className="relative flex flex-col gap-2">
```

Comment:
`<a href={url} rel="nofollow noreferrer" className="relative flex flex-col gap-2" target="_blank">`
Let's open it in a new tab. You know, conversion rates 📈 and time on page 📉. We don't want people accidentally leaving the website :d